docs/src/components/Example.tsx

```diff
@@ -1,13 +1,20 @@
 import { View } from '@aws-amplify/ui-react';
+import React from 'react';
```

Comment:
I understand that we're moving away from default imports, right? can we start with this one?
```suggestion
import * as React from 'react';
```